challenges/minimum-coin-change/src/csharp/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("MinimumCoinChallenge.Tests")]
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2, PublicKey=0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7")]
```

Comment:
Technically, I don't need this, as I'm not mocking anything yet.  That said, I always seem to forget it when I do, so I'm fine with the small bit of added clutter here.

Reply:
Changed my mind and decided to remove this, since it's not likely that mocking will be needed here.